libs/math/include/math/fundamental_operators.hpp

```diff
@@ -332,11 +332,8 @@ meta::IfIsMathArray<ArrayType, void> Add(ArrayType const &array1, ArrayType cons
   }
   else
   {
-    // TODO(issue 1257): Implement const version of Tensor::Broadcast
-    ArrayType a = array1.Copy();
-    ArrayType b = array2.Copy();
-    if (!(Broadcast([](typename ArrayType::Type x, typename ArrayType::Type y) { return x + y; }, a,
-                    b, ret)))
+    if (!(Broadcast([](typename ArrayType::Type x, typename ArrayType::Type y) { return x + y; },
```

Comment:
awesome!

I reckon on Friday we should have a day where we go through closing old issues